.github/workflows/deploy-development.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           port: ${{ secrets.SSH_PORT }}
           script: |
             cd ~/megabot-dev
+            git reset --hard
```

Comment:
I mainly added this because I had to modify code in the server itself. So this is a tiny bit of future proofing so it resets before pulling